### PR TITLE
Hotkeys - ALT == Option key 4 mac added

### PR DESCRIPTION
**Pull request by metanoder**
Hotkeys: Added ALT == Option key compatibility for mac

### CODE REVIEW

File: addons/dialogic/Editor/TimelineEditor/VisualEditor/timeline_editor_visual.gd [hunk 1]
```diff
@@ -346,21 +346,23 @@ func _input(event:InputEvent) -> void:
 			_add_event_button_pressed(DialogicTextEvent.new(), true)
 			get_viewport().set_input_as_handled()
 		
-		"Ctrl+Shift+T", "Ctrl+Alt+T": # Add text event with current or previous character 
+		"Ctrl+Shift+T", "Ctrl+Alt+T", "Ctrl+Option+T": # Add text event with current or previous character 
 			get_viewport().set_input_as_handled()
 			var ev := DialogicTextEvent.new()
-			ev.character = get_previous_character(event.as_text() == "Ctrl+Alt+T")
+			ev.character = get_previous_character(event.as_text() == "Ctrl+Alt+T" || "Ctrl+Option+T")
```
Comment (Jowan-Spooner) [inline addons/dialogic/Editor/TimelineEditor/VisualEditor/timeline_editor_visual.gd:352]: Could you use or here. Following the styleguide we try to not use the unrecommended aliases.

**Comment by Jowan-Spooner** [comment]
Thanks. Yeah idk, that's what they are called [here](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#operators). We use ! a lot for not but have tried to avoid the other ones. Don't ask me 🤷 

Is this ready for merging, or do you want to test the other shortcuts as well?

**Comment by metanoder** [comment]
> Thanks. Yeah idk, that's what they are called [here](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#operators). We use ! a lot for not but have tried to avoid the other ones. Don't ask me 🤷
> 
> Is this ready for merging, or do you want to test the other shortcuts as well?

;) Yea, just a pseudo developer here.  lol.  

Found that the up and down keys also use alt.  so added this to the pull request.  Otherwise, all the other ones work.  Great stuff as always @Jowan-Spooner .  Appreciate you.  Yep.  this small contribution is ready to merge.

**Comment by Jowan-Spooner** [comment]
Thanks!

**Pull request merged**